storage_scrubber/src/scan_pageserver_metadata.rs

```diff
@@ -306,15 +317,18 @@ pub async fn scan_pageserver_metadata(
         tenant_timeline_results.push((ttid, data));
     }
 
+    let tenant_id = tenant_id.expect("Must be set if results are present");
+
     if !tenant_timeline_results.is_empty() {
         analyze_tenant(
             &remote_client,
-            tenant_id.expect("Must be set if results are present"),
+            tenant_id,
             &mut summary,
             tenant_objects,
             tenant_timeline_results,
             highest_shard_count,
         )
+        .instrument(info_span!("", tenant = %tenant_id))
```

Comment:
should assign each of the span a name

Reply:
added a name to both spans: bb1de166adfa49f8f1d495496486d6bcbde51103